RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -87,7 +87,7 @@
     minPt = 0.2,
 )
 highPtTripletStepTrajectoryFilterBase = _highPtTripletStepTrajectoryFilterBase.clone(
-    maxCCCLostHits = 2,
+    maxCCCLostHits = 0,
     minGoodStripCharge = dict(refToPSet_ = 'SiStripClusterChargeCutLoose')
```

Comment:
Not intended (slipped through). The cut of estimator should be at least as high as of the filter. I'll file a fix after the next IB.